yt/visualization/plot_window.py

```diff
@@ -1267,6 +1269,73 @@ def show_axes(self, field=None):
             self.plots[f].show_axes()
         return self
 
+    def export_to_mpl_figure(self, nrows_ncols, axes_pad=1.0,
+                             label_mode="L",
+                             cbar_location="right", cbar_size="5%",
+                             cbar_mode="each", cbar_pad="0%"):
+        r"""
+        Creates a matplotlib figure object with the specified axes arrangment, nrows_ncols,
+        and maps the underlying figures to the matplotlib axes.
+
+        The return is a matplotlib figure object.
+
+        Parameters
+        ----------
+
+        nrows_ncols : tuple
+           the number of rows and columns of the axis grid (e.g., nrows_ncols=(2,2,))
+        axes_pad : float
+           padding between axes in inches
+        label_mode : one of "L", "1", "all"
+           arrangement of axes that are labeled
+        cbar_location : one of "left", "right", "bottom", "top"
+           where to place the colorbar
+        cbar_size : string (percentage)
+           scaling of the colorbar (e.g., "5%")
+        cbar_mode : one of "each", "single", "edge", None
+           how to represent the colorbar
+        cbar_pad : string (percentage)
+           padding between the axis and colorbar (e.g. "5%")
```

Comment:
I'm okay with duplicating matplotlib's documentation, but I'd suggest to mention somewhere that all those arguments are to be directly fed to `mpl_toolkits.axes_grid1.ImageGrid`.

Essentially I suggest you reprise the existing comment from the cookbook example you're adapting.